src/native-common/WebView.tsx

```diff
@@ -41,6 +41,7 @@ export class WebView extends RX.WebView<{}> {
                 scalesPageToFit={ this.props.scalesPageToFit }
                 onError={ this.props.onError }
                 onLoadStart={ this.props.onLoadStart }
+                onMessage={ this.props.onMessage }
```

Comment:
If we're going to support onMessage, then this needs to be added to WebViewProps in Types.ts. And a corresponding implementation is needed for the web implementation of WebView.tsx.